SPEC.md

```diff
@@ -281,9 +281,11 @@ keys: https://example.com/pubkeys.gpg
 
 This mechanism is only used for discovery of contents URLs.
 
-If the first attempt at fetching the discovery URL returns a status code other than `200 OK` or does not contain any `ac-discovery` meta tags then the next higher path in the name should be tried.
+If the first attempt at fetching the discovery URL returns a status code other than `200 OK`, 3xx, or does not contain any `ac-discovery` meta tags then the next higher path in the name should be tried.
```

Comment:
I'm following `xx` convention from the [RFC](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html)